index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Portfolio</title>
+	<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
+	<link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+	<link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+	<div class="app-container">
+		<nav class="app-nav">
+			<a id="logo" href="#">My logo</a>
+			<i id="menu-icon" class="bi bi-list"></i>
+		</nav>
+		<main class="app-greeting">
+			<h1>I'm Sane great to see you!</h1>
+			<p>I'm a software developer! I can help you build a product, feature or website. Look through some of my work
+				and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+			<p id="lets-connect">LET'S CONNECT</p>
+			<ul class="socials">
+				<li><a href="#"><img src="/images/twitter.svg" alt="Twitter logo"></a></li>
+				<li><a href="#"><img src="/images/linkedin.svg" alt="Linkedin Logo"></a></li>
+				<li><a href="#"><img src="/images/medium.svg" alt="Medium Logo"></a></li>
+				<li><a href="#"><img src="/images/github.svg" alt="Github Logo"></a></li>
+				<li><a href="#"><img src="/images/angellist.svg" alt="Angellist Logo"></a></li>
```

Comment:
- [ ] Kindly ensure that the whole of this code renders on the UI
## This what you UI looks like
![image](https://user-images.githubusercontent.com/86200356/192078362-198274cc-886b-4cc2-b179-5e95d88f3de1.png)
## Figma design
![image](https://user-images.githubusercontent.com/86200356/192078421-aaf85181-4745-476c-b2de-ed2e085f7432.png)